<launch>
       <!-- Turtlesim Node-->
      <node pkg="turtlesim" type="turtlesim_node" name="sim"/>

      <node pkg="turtlesim" type="turtle_teleop_key" name="teleop" output="screen"/>

      <!-- Axes -->
      <param name="scale_linear" value="2" type="double"/>
      <param name="scale_angular" value="2" type="double"/>
      <param name="radius" value="6.0" type="double"/>
      <param name="direction_of_rotation" value="-1" type="double"/>

      <node pkg="learning_tf2" type="turtle_tf2_broadcaster"
            args="/turtle1" name="turtle1_tf2_broadcaster" />
      <node pkg="learning_tf2" type="turtle_tf2_broadcaster"
            args="/turtle2" name="turtle2_tf2_broadcaster" />
      <node pkg="learning_tf2" type="turtle_tf2_broadcaster_carrot" name="turtle_tf2_broadcaster_carrot" />

      <node pkg="learning_tf2" type="turtle_tf2_listener"
            name="listener" />

      <node pkg="rviz" type="rviz" name="rviz" args="-d $(find learning_tf2)/rviz/carrot.rviz"/>
  </launch>